bcda/models/alr.go

```diff
@@ -23,3 +24,17 @@ type AlrMetaData struct {
 	ACO       string
 	Timestamp time.Time
 }
+
+// Data Structure for Jobs
+type AlrJobs struct {
+	ID     uint
+	ACO    string
```

Comment:
What do you think about matching this with our current pattern of CMSID? We would change this struct and the AlrMetaData one as well to match up.